float64[] x # a list of x coordinate of circle center x
float64[] y # a list of y coordinate of circle center y
float64[] radius
int64[] id # id of this landmark (data association)